jpo-geojsonconverter/src/main/java/us/dot/its/jpo/geojsonconverter/partitioner/RsuTimKey.java

```diff
@@ -0,0 +1,46 @@
+package us.dot.its.jpo.geojsonconverter.partitioner;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+/**
+ * Kafka key for TIM messages. Partition on RSU ID, with TIM-specific fields for better message organization.
+ */
+@Data
+@NoArgsConstructor
+public class RsuTimKey implements RsuIdKey {
+
+    private String rsuId;
+    private String packetId;
+    private Integer msgCnt;
```

Comment:
question: Are `packetId` and `msgCnt` likely to stay the same, so that related TIMs would be on the same partition?  If not would it be better to just use rsuId to partition on?